src/components/common/TextArea.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+const TextArea = () => {
+
+	React.useEffect(() => 
+		{
+			document.addEventListener('keydown', () => {
+				setTimeout(calculateHeight, 0);
+			})
+		}
+	,[]);
+
+	const [height, setHeight]= React.useState<number>(36);
```

Comment:
```suggestion
        const INITIAL_HEIGHT = 36

	const [height, setHeight]= React.useState<number>(INITIAL_HEIGHT);
```

Avoid hardcoding values, use constants instead.